backend/test/integration/chatController.test.ts

```diff
@@ -24,7 +24,8 @@ declare module 'express-session' {
 }
 
 // mock the api call
-const mockCreateChatCompletion = jest.fn();
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const mockCreateChatCompletion = jest.fn<any>();
```

Comment:
Why do you need to use an any here, when there's a jest.fn() call below that doesn't specify a generic type?

Reply:
If we do not set a type then each time we want to mock its behaviour we get an error:

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/02a49a70-38bf-4035-8769-b69dc5b7466e)

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/a7a0a75d-b57d-4a62-a6cc-008bcf377c38)

Reply:
Using `any` is not helping either, cos it tells us nothing about how the mock is expected to be used (and forces us to disable a perfectly good linter rule). What values are we expecting to be returned from createChatCompletion? Can we not provide a union type to the function for that?

Reply:
indeed. I've removed all the any types now!